### DISCUSSION TITLE
"Failed workflow run" notifications open in browser rather than in app

### DISCUSSION THREAD

**Discussion by Starlight220** [community/community#41007] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Tapping on a "failed workflow run" notification (in the "notifications" tab of the app) opens `github.com/<owner>/<repo>/actions/runs/<run-id>` in a browser.
However, navigating to the workflow run log through the checks section of a PR works opens correctly in the app.

Tested on Android 8, with both stable and beta versions of the app.

**Comment by candyho** [comment]
Thanks for reporting this! I will defer this to the team and hopefully we can work on a fix soon!

**Reply by Danlachapelle** [reply]
> Thanks for reporting this! I will defer this to the team and hopefully we can work on a fix soon!

**Reply by metecanpolat1** [reply]
[](https://github.com/stripe-samples/placing-a-hold/issues/15)

**Reply by Danlachapelle** [reply]
> [](https://github.com/stripe-samples/placing-a-hold/issues/15)

**Comment by Starlight220** [comment, marked as answer]
Appears to be fixed now; thanks!

**Reply by Untitled-800** [reply]
Oh goood))0)

**Comment by ariftechloy** [comment]
Yes
